src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs

```diff
@@ -265,6 +265,15 @@ protected override async Task HandleChallengeAsync(AuthenticationProperties prop
             }
         }
 
+        protected override Task HandleForbiddenAsync(AuthenticationProperties properties)
+        {
+            var authenticationFailedContext = new AuthenticationFailedContext(Context, Scheme, Options)
```

Comment:
Don't re-use AuthenticationFailedContext, create a new context type.

Reply:
I have created a new context as suggested and applied this.